### DISCUSSION TITLE
Getting a never before seen problem: Event hash does not match any of your saved devices.

### DISCUSSION THREAD

**Discussion by jjsenay** [ambianic/ambianic-ui#794] [Q&A] [answered]
All

Been having some real ruff weather here in the midwest.  Tried to pull down a time line for the Edge in my garage.

Get this notification on screen:

![Screen Shot 2021-12-11 at 9 15 15 PM](https://user-images.githubusercontent.com/8701163/145699217-d919e114-224a-449e-8852-2173f8ae46a7.png)

**Comment by jjsenay** [comment]
Never seen this before.  Trying to connect

![Screen Shot 2021-12-11 at 9 36 16 PM](https://user-images.githubusercontent.com/8701163/145699256-2b777c69-15aa-4e21-ac20-0be15110ae39.png)

**Comment by jjsenay** [comment]
Click on my devices above and I get :
![Screen Shot 2021-12-11 at 9 44 34 PM](https://user-images.githubusercontent.com/8701163/145699399-b83ab4ab-90d9-4fea-b09b-357a7ce9e37c.png)

This has never been an address for any of my Edge devices.

**Comment by jjsenay** [comment]
854... is not on my internal network

**Comment by jjsenay** [comment]
Don't know what is going on

**Comment by ivelin** [comment, marked as answer]
> All
> 
> Been having some real ruff weather here in the midwest. Tried to pull down a time line for the Edge in my garage.
> 
> Get this notification on screen:
> 
> ![Screen Shot 2021-12-11 at 9 15 15 PM](https://user-images.githubusercontent.com/8701163/145699217-d919e114-224a-449e-8852-2173f8ae46a7.png)

You are probably opening the event URL in a browser that doesn't know about  your devices. See the [Troubleshooting section](https://docs.ambianic.ai/users/notifications/#troubleshooting) in the docs.
